src/features/broker-protection/execute.js

```diff
@@ -0,0 +1,31 @@
+import { click } from './click.js'
+import { ErrorResponse } from './types.js'
+
+/**
+ * @param {object} action
+ * @param {string} action.id
+ * @param {"extract" | "fillForm" | "click" | "expectation" | "getCaptchaInfo" | "solveCaptcha" | "navigate"} action.actionType
+ * @param {any} data
+ * @return {import('./types.js').ActionResponse}
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function execute (action, data) {
```

Comment:
should this return a Promise?

Reply:
no - originally all actions were async, but it created a lot of noise and in the end I realised none were currently doing anything async.

So, I've made them all sync as a default, and we can always add `async` ones when needed :)